Adapter.C

```diff
@@ -203,6 +203,12 @@ bool preciceAdapter::Adapter::configFileRead()
     }
     DEBUG(adapterInfo("    FSI module enabled : " + std::to_string(FSIenabled_)));
 
+    if (adapterConfig_["provideMeshConnectivity"])
```

Comment:
But this has nothing to do with the patches, right?

Reply:
Ok this is done and it is associated to the locationsType, since connectivity only makes sense in case of faceNodes.

Reply:
This means, you can choose the option for each interface, but in case of faceCenters, the value is ignored and a warning is raised. Hence, the user can only choose in case of faceNodes